articles/azure-functions/functions-bindings-service-bus.md

```diff
@@ -508,7 +508,7 @@ public String pushToQueue(
  }
 ```
 
- In the [Java functions runtime library](/java/api/overview/azure/functions/runtime), use the `@QueueOutput` annotation on function parameters whose value would be written to a Service Bus queue.  The parameter type should be `OutputBinding<T>`, where T is any native Java type of a POJO.
+ In the [Java functions runtime library](/java/api/overview/azure/functions/runtime), use the `@ServiceBusQueueOutput` annotation on function parameters whose value would be written to a Service Bus queue.  The parameter type should be `OutputBinding<T>`, where T is any native Java type of a POJO.
```

Comment:
Just taking a guess based on the `@ServiceBusTopicOutput`

Reply:
Thanks for the heads-up. I am working on the Java and Python code samples in another PR, so I will validate before publishing 😉